apps/explorer/lib/explorer/exchange_rates/source.ex

```diff
@@ -56,17 +57,63 @@ defmodule Explorer.ExchangeRates.Source do
     end
   end
 
+  defp remove_market_cap_if_certain_token(value, address) do
+    if address in ["0x49d5c2bdffac6ce2bfdb6640f4f80f226bc10bab", "0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7"] do
```

Comment:
What is special about these 2 addresses?

Reply:
The wrapped ethereum and wrapped avax token don't have a defined market  and coingecko return market cap equal to zero, to avoid displaying a circulating market cap of zero we remove the field and save nill on database. This result in a value of the circulating market cap equal to `N/A` (no data) instead of 0 on the blockscout UI.
The code reported is now outdated and we use env variable instead of hardcoded addresses, morerover the check is now done on the market cap field and not on the address (if circulating market cap equal to zero remove the field)